package.json

```diff
@@ -20,7 +20,8 @@
     "i18n:validate": "ttag validate i18n/zh_TW.po",
     "clean": "run-s clean:*",
     "clean:server": "rm -rf build",
-    "clean:liff": "rm -rf liff"
+    "clean:liff": "rm -rf liff",
+    "notify": "babel-node -r dotenv/config build/scripts/scanRepliesAndNotify.js"
```

Comment:
When using `babel-node`, we should be able to run from `src/script` directly, or there may be error when it's not built yet

Reply:
Sorry, I forgot to change the script path..